vllm_spyre/config/supported_configurations.yaml

```diff
@@ -0,0 +1,85 @@
+      { platform: "x86_64", cb: True,  tp_size: 1, max_model_len: 128, max_num_seqs: 4, num_blocks:  0 },
+      { platform: "x86_64", cb: True,  tp_size: 1, max_model_len: 128, max_num_seqs: 4, num_blocks:  4 },
+      { platform: "x86_64", cb: True,  tp_size: 1, max_model_len: 128, max_num_seqs: 4, num_blocks:  8 },
+      { platform: "x86_64", cb: True,  tp_size: 1, max_model_len: 192, max_num_seqs: 2, num_blocks: 16 },
+      { platform: "x86_64", cb: True,  tp_size: 1, max_model_len: 192, max_num_seqs: 3, num_blocks: 16 },
+      { platform: "x86_64", cb: True,  tp_size: 1, max_model_len: 256, max_num_seqs: 4, num_blocks:  0 },
+      { platform: "x86_64", cb: True,  tp_size: 2, max_model_len: 256, max_num_seqs: 4, num_blocks:  0 },
+      { platform: "x86_64", cb: True,  tp_size: 4, max_model_len: 256, max_num_seqs: 4, num_blocks:  0 },
+      { platform: "x86_64", cb: True,  tp_size: 8, max_model_len: 256, max_num_seqs: 4, num_blocks:  0 },
+    ]
+  - model: "ibm-ai-platform/micro-g3.3-8b-instruct-1b-FP8"
+    configs: [
+      { platform: "x86_64", cb: False, tp_size: 1, warmup_shapes: [[64, 20, 4], [128, 20, 2]] },
+      { platform: "x86_64", cb: False, tp_size: 2, warmup_shapes: [[64, 20, 4]] },
+      { platform: "x86_64", cb: False, tp_size: 4, warmup_shapes: [[64, 20, 4]] },
+      { platform: "x86_64", cb: False, tp_size: 8, warmup_shapes: [[64, 20, 4]] },
+      { platform: "x86_64", cb: True,  tp_size: 1, max_model_len: 128, max_num_seqs: 2, num_blocks: 4 },
+      { platform: "x86_64", cb: True,  tp_size: 1, max_model_len: 128, max_num_seqs: 2, num_blocks: 8 },
+      { platform: "x86_64", cb: True,  tp_size: 1, max_model_len: 128, max_num_seqs: 4, num_blocks: 4 },
+      { platform: "x86_64", cb: True,  tp_size: 1, max_model_len: 128, max_num_seqs: 4, num_blocks: 8 },
+      { platform: "x86_64", cb: True,  tp_size: 1, max_model_len: 192, max_num_seqs: 2, num_blocks: 16 },
+      { platform: "x86_64", cb: True,  tp_size: 1, max_model_len: 192, max_num_seqs: 3, num_blocks: 16 },
+      { platform: "x86_64", cb: True,  tp_size: 1, max_model_len: 256, max_num_seqs: 4, num_blocks: 0 },
+      { platform: "x86_64", cb: True,  tp_size: 2, max_model_len: 256, max_num_seqs: 4, num_blocks: 0 },
+      { platform: "x86_64", cb: True,  tp_size: 4, max_model_len: 256, max_num_seqs: 4, num_blocks: 0 },
+      { platform: "x86_64", cb: True,  tp_size: 8, max_model_len: 256, max_num_seqs: 4, num_blocks: 0 },
+    ]
+  - model: "ibm-granite/granite-3.3-8b-instruct"
+    configs: [
+      { platform: "x86_64",  cb: True,  tp_size: 4, max_model_len: 2048, max_num_seqs: 4 },
+      { platform: "x86_64",  cb: True,  tp_size: 4, max_model_len: 4096, max_num_seqs: 4 },
+      { platform: "x86_64",  cb: True,  tp_size: 4, max_model_len: 8192, max_num_seqs: 4 },
+      { platform: "s390x",   cb: True,  tp_size: 4, max_model_len: 8192, max_num_seqs: 4 },
+      { platform: "s390x",   cb: False, tp_size: 4, warmup_shapes: [[7168, 1024,  1]] },
```

Comment:
let's not publicly document Z and power support for now- there are no public builds for them.

For the simple case, let's start by running this validation while ignoring the platform flag